web/src/beta/features/Editor/Visualizer/hooks.ts

```diff
@@ -87,7 +92,16 @@ export default ({
     [selected],
   );
 
-  const layers = useMemo(() => processLayers(nlsLayers), [nlsLayers]);
+  const layers = useMemo(() => {
+    const processedLayers = processLayers(nlsLayers);
+    if (showStoryPanel) {
+      return processedLayers;
+    }
+    return processedLayers?.map(layer => ({
+      ...layer,
+      hidden: !currentPage?.layersIds?.includes(layer.id),
```

Comment:
Why do we need `showStoryPanel`? Can't we use just `hidden` property?

Reply:
@keiya01 sure you are correct I will remove it

Reply:
Sorry incorrect (check slack for answer)